gradle/gradle-publish.gradle

```diff
@@ -46,9 +46,14 @@ afterEvaluate {
         }
     }
 
+    def isCiEnvironment = System.getenv('CI')
```

Comment:
Since we publish through CI will this remove signing on our CI publishing workflow?

Reply:
It shouldn't, but publishing it is probably the only way to test it.
The `setRequried` block should make sure that signing happens.
See : https://docs.gradle.org/current/javadoc/org/gradle/plugins/signing/SigningExtension.html#setRequired-boolean-